DSESwift/DatabaseHandler.swift

```diff
@@ -465,8 +466,15 @@ class DatabaseHandler: NSObject{
         return (startTime != nil) && (endTime != nil) && (startTime?.timeIntervalSince1970 >= endTime?.timeIntervalSince1970)
     }
     
-    private static func getUserId() -> String {
-        return KeychainWrapper.stringForKey(KEYCHAIN_USERID)!
+    private static func getUserId() throws -> String {
+        let defaults = NSUserDefaults.standardUserDefaults()
```

Comment:
@kanekin what is the reasoning to use NSUserDefaults instead of KeychainWrapper?